src/analyzing/Eval_il_partial.ml

```diff
@@ -230,6 +230,7 @@ let rec eval (env : G.svalue Dataflow_var_env.t) (exp : IL.exp) : G.svalue =
   match exp.e with
   | Fetch lval -> eval_lval env lval
   | Literal li -> G.Lit li
+  | Cast (_, e) -> eval env e
```

Comment:
Out of caution, I'd prefer this is restricted to TS for now. I think TS is kind of a special case. Even in Java you would need to do something when casting e.g. `(double)42`. That would mean passing the language to `eval`, it's a bit more work, but this can be useful later too.

Reply:
If you rebase, I just merged another PR that adds a field `lang : Lang.t` to the `env` here.